.travis/install.sh

```diff
@@ -9,10 +9,8 @@ if [[ "$(uname -s)" == 'Darwin' ]]; then
     export HOMEBREW_TEMP=~/brew-temp
     mkdir $HOMEBREW_LOGS
     mkdir $HOMEBREW_TEMP
-    # brew update
-    if [[ "${OPENSSL}" != "0.9.8" ]]; then
-        brew outdated openssl || brew upgrade openssl
-    fi
+
+    brew install rbenv/tap/openssl@1.0
```

Comment:
hmm, can you add a comment why this is like it is (compared to using something like `openssl@1.0`?

Reply:
Added a note. More details in the tap docs https://docs.brew.sh/Taps#the-brew-tap-command

tldr; rbenv/tap becomes github.com/rbenv/homebrew-tap and openssl Formula is inside that repo

Reply:
thanks for the note, will merge after tests.